src/patternfly/components/Card/card.scss

```diff
@@ -22,6 +22,11 @@
   --pf-c-card--child--PaddingRight: var(--pf-global--spacer--lg);
   --pf-c-card--child--PaddingBottom: var(--pf-global--spacer--lg);
   --pf-c-card--child--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-card__header-toggle--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
+  --pf-c-card__header-toggle--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-card__header-toggle--MarginBottom: calc(var(--pf-global--spacer--form-element) * -1);
+  --pf-c-card__header-toggle--MarginLeft: calc(var(--pf-global--spacer--md) * -1);
+  --pf-c-card--m-expanded__header-toggle-icon--Rotate: 90deg;
   --pf-c-card__title--FontSize: var(--pf-global--FontSize--md);
```

Comment:
--pf-c-card__header-toggle-icon--Transition: var(--pf-global--Transition);

Reply:
Good call, I forgot to add that.